wire-tap/wire-tap.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/http;
+
+type SnowflakeRequest record {
+    string statement;
+    string timeout = "60";
+    string database = "messageLog";
+    string schema = "message";
+    string role = "logger";
+};
+
+type WireTapRequest record {|
+    string 'table;
+    "INFO"|"WARNING"|"ERROR" severity;
+    string message;
+|};
+
+type StockResponse record {
+    string parentHandlingUnitUUID;
+    string StockItemUUID;
+    string EWMWarehouse;
+    string HandlingUnitNumber;
+    string ShelfLifeExpirationDate;
+    string CountryOfOrigin;
+};
+
+http:Client wh = check new("http://api.sap.com.balmock.io");
+
+service /warehouse on new http:Listener(8080) {
+    
+    resource function get stock(string parentId, string productId) returns StockResponse|error {
+        StockResponse result = check wh->/WarehouseAvailableStock/[parentId]/[productId];
```

Comment:
Use hyphens is better.
Also the calling resource should be in plural form.
```suggestion
        StockResponse result = check wh->/warehouse\-available\-stocks/[parentId]/[productId];
```

Reply:
This is the path used with the endpoint, so we should stick to that right?